docs/PROJECT_SCOPING.md

```diff
@@ -0,0 +1,61 @@
+# Project Scoping
+
+Delphi codebases can often have indistinct boundaries, with common source files
+being shared by multiple Delphi projects. There are a few ways of handling this from
+a SonarDelphi perspective.
+
+## Method 1 - Having a different Sonar project for each `.dproj` file
+
+This is the ideal project setup from an analysis quality standpoint.
+
+Obviously, only do this for significantly different `.dproj` files - if your project has several
+almost identical `.dproj` files to support multiple Delphi versions, for example, then it's
+probably best to have a single project that includes only your latest `.dproj`.
+
+## Method 2 - Having one big Sonar project with multiple `.dproj` files in it
+
+If SonarDelphi encounters multiple `.dproj` files during analysis, it will simply aggregate the values
+from all of them together. In practice, this means that for every source file SonarDelphi would use every
+search path and compiler define from the included `.dproj` files.
+
+This is almost never the right thing to do, because conflicting configuration between `.dproj` files
+usually renders the scan results unreliable.
+
+Take this example:
+
+```
+MyDelphiCode/
+  ProjectA/
+    ProjectA.dproj
+  ProjectB/
+    ProjectB.dproj
+  Common/
+    CommonFile.pas
+```
+
+If `ProjectA.dproj` and `ProjectB.dproj` both include `CommonFile.pas`, there
+are now two "versions" of `CommonFile.pas`: one that uses the search path, configuration,
+and compiler defines of `ProjectA.dproj`, and another that uses those of `ProjectB.dproj`.
+
+How can SonarDelphi reasonably come up with a "correct" interpretation of `CommonFile.pas` to analyze?
+As a fallback, it makes a best-effort attempt by simply combining the settings from both `.dproj` files
+together. Unless both `.dproj` files are set up very similarly, this is rarely the desired outcome.
+
+In this case, `ProjectA` and `ProjectB` should be two separate Sonar projects. `CommonFile.pas` could be
+handled in a few different ways:
+
+* **Included in both Sonar projects:** Gets both interpretations, but results in issue duplication between
```

Comment:
Maybe this could be a table?
I don't know, but the bulleted list with all of this bold feels like an eyesore.